tensorflow_lite_support/ios/task/audio/sources/TFLAudioClassifier.h

```diff
@@ -0,0 +1,175 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import <Foundation/Foundation.h>
+
+#import "tensorflow_lite_support/ios/task/audio/core/audio_tensor/sources/TFLAudioTensor.h"
+#import "tensorflow_lite_support/ios/task/core/sources/TFLBaseOptions.h"
+#import "tensorflow_lite_support/ios/task/processor/sources/TFLClassificationOptions.h"
+#import "tensorflow_lite_support/ios/task/processor/sources/TFLClassificationResult.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Options to configure `TFLAudioClassifier`.
+ */
+NS_SWIFT_NAME(AudioClassifierOptions)
+@interface TFLAudioClassifierOptions : NSObject
+
+/**
+ * Base options that are used for creation of any type of task.
+ * @discussion Please see `TFLBaseOptions` for more details.
+ */
+@property(nonatomic, copy) TFLBaseOptions *baseOptions;
+
+/**
+ * Options that configure the display and filtering of results.
+ * @discussion Please see `TFLClassificationOptions` for more details.
+ */
+@property(nonatomic, copy) TFLClassificationOptions *classificationOptions;
+
+/**
+ * Initializes a new `TFLAudioClassifierOptions` with the absolute path to the model file stored
+ * locally on the device.
+ *
+ * @discussion The external model file, must be a single standalone TFLite file. It could be packed
+ * with TFLite Model Metadata[1] and associated files if exist. Fail to provide the necessary
+ * metadata and associated files might result in errors. Check the
+ * [documentation](https://www.tensorflow.org/lite/convert/metadata) for each task about the
+ * specific requirement.
+ *
+ * @param modelPath An absolute path to a TensorFlow Lite model file stored locally on the device.
+ *
+ * @return An instance of `TFLAudioClassifierOptions` initialized to the given model path.
+ */
+- (instancetype)initWithModelPath:(NSString *)modelPath;
+
+- (instancetype)init NS_UNAVAILABLE;
```

Comment:
There's no need to mark these `init` and `new` method as NS_UNAVAILABLE. Let's align with the TFLImageClassifier API.

Reply:
This doesn't seem to be resolved.

Reply:
It i resolved . Can you follow [this link](https://github.com/tensorflow/tflite-support/pull/835/files#diff-08fc61a126ede0c4b4e8476e20a6916e3641bfcc7426aeb01a3852ee1395c6d4R56) and check again. I think the link above this comment shows an outdated version.